actionpack/lib/action_dispatch/middleware/cookies.rb

```diff
@@ -83,7 +83,10 @@ def cookies_digest
   #   cookies[:lat_lon] = JSON.generate([47.68, -122.37])
   #
   #   # Sets a cookie that expires in 1 hour.
-  #   cookies[:login] = { value: "XJ-122", expires: 1.hour.from_now }
+  #   cookies[:login] = { value: "XJ-122", expires: 1.hour }
+  #
+  #   # Sets a cookie that expires at a specific time
+  #   cookies[:oreo] = { value: "chocolate cookies", expires: Time.utc(2020, 10, 15, 5) }
```

Comment:
I'm not sure if we're allowed to reference a trademark in documentation/changelogs. While cute, I'd avoid it.

Reply:
:sweat_smile: I was inspired by the Python community. I'll change that! :smile: